src/stories/epics/EpicTable.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+import { EpicTable } from './EpicTable.stories';
+
+<Meta title="Epics/Table/Intro" component={EpicTable} />
+
+## Overview
+
+**Epics** are bodies of work that are large enough to be broken down to a number
+of smaller **tasks** within a given **project**.
+
+The **EpicTable** renders an [SLDS
+DataTable](https://react.lightningdesignsystem.com/components/data-tables/)
+accepting a collection of `epics` to render in the table. This allows users to
+view a list of epics and their related information created within a given
+project.
+
+## Sub components
```

Comment:
I guess we will have a better idea of which headings to have as we write documentation for more of these. Should "Overview" be on on every component and then all other subheadings vary depending on the component? Or should we have the same subheadings on every component? Still figuring this out. . .

Reply:
I think it can be a little flexible @dvdherron. Using "Overview" makes sense for all so I like that idea and the subheadings can be the same if they exist, and display nothing if they don't.